cni/nnsgrpcclient/nnsgrpclient.go

```diff
@@ -0,0 +1,89 @@
+// Node network service grpc client
+package nnsgrpcclient
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-container-networking/log"
+	nns "github.com/Azure/azure-container-networking/proto/nodenetworkservice/3.302.0.744"
+	"google.golang.org/grpc"
+	"time"
+)
+
+const (
+	nnsPort = "6668"
+	apiTimeout = 5 * time.Minute // recommended timeout from node service
```

Comment:
how intensive are these operations that they may require up to 5 minutes? 😁

Reply:
This was recommended by nns team to us. I don't think API is intensive or takes long time. When it works, whole thing gets done in matter of seconds. But so far we have only tested it on a single node with single pod being created. At a scale, it is possible they NNS might take longer. Scale testing hasn't be done yet.